docs/releases/5.2.3.txt

```diff
@@ -9,4 +9,6 @@ Django 5.2.3 fixes several bugs in 5.2.2.
 Bugfixes
 ========
 
-* ...
+* Fixed a regression in Django 5.2 that caused ``bulk_update()`` on a model
+  with a ``JSONField`` to save ``None`` as a JSON ``null`` instead of SQL
+  ``NULL`` (:ticket:`36440`).
```

Comment:
The impact is probably much wider than this, but I haven't got any other concrete cases to report about. I am not sure about the impact of skipping any of the other `get_db_prep_save()` methods in Django core.